From 7dee32fa2c68bd0e85253a6b31b4ed5ce829c6dc Mon Sep 17 00:00:00 2001
From: Sam Yang <r52096@freescale.com>
Date: Tue, 5 May 2009 11:31:51 +0800
Subject: [PATCH] ENGR00112155 Error bool definition in ipu.h for user space c++ compiler

When compiling android opencore, a redeclared compiling error occur because
of bool definition in ipu.h.

Signed-off-by:Sam Yang <r52096@freescale.com>
---
 include/linux/ipu.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/linux/ipu.h b/include/linux/ipu.h
index 0857e18..d76602a 100644
--- a/include/linux/ipu.h
+++ b/include/linux/ipu.h
@@ -29,7 +29,9 @@
 #ifdef __KERNEL__
 #include <linux/interrupt.h>
 #else
-#define bool char
+#ifndef __cplusplus
+typedef unsigned char bool;
+#endif
 #define irqreturn_t int
 #define dma_addr_t int
 #define u32 unsigned int
-- 
1.5.4.4

